docs/topics/mta-web-creating-migration-waves.adoc

```diff
@@ -6,18 +6,18 @@
 [id="mta-web-creating-migration-waves_{context}"]
 = Creating a migration wave
 
-By creating a migration wave, you can group applications that you want to migrate on a given schedule. You can also track each migration by exporting a list of the wave's applications to the Jira issue management system. This automatically creates a separate Jira issue for each application of the migration wave.
+A migration wave is a group applications that you can migrate on a given schedule. You can track each migration by exporting a list of the wave's applications to the Jira issue management system. This automatically creates a separate Jira issue for each application of the migration wave.
 
 .Procedure
 
 . In the *Migration* view, click *Migration waves*.
 . Click *Create new*. The *New migration wave* window opens.
-. Enter the following information or select from drop-down lists:
-* Name (optional). If the name is not given, you can use the start and end dates to identify migration waves.
-* Potential start date. This date must be later than the current date.
-* Potential end date. This date must be later than the start date.
-* Stakeholders (optional)
-* Stakeholder groups (optional)
+. Enter the following information:
+* *Name* (optional). If the name is not given, you can use the start and end dates to identify migration waves.
+* *Potential start date*. This date must be later than the current date.
+* *Potential end date*. This date must be later than the start date.
+* *Stakeholders* (optional)
+* *Stakeholder groups* (optional)
```

Comment:
Both fields accept one or more values in case you want to specify it

Reply:
Oki, will take it into consideration, thank you :-)